src/components/map/MapService.js

```diff
@@ -19,10 +19,16 @@ goog.require('ga_urlutils_service');
 
   module.provider('gaTileGrid', function() {
     var origin = [420000, 350000];
-    var defaultResolutions = [4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250,
-        2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5,
-        2.5, 2, 1.5, 1, 0.5];
-    var wmsResolutions = defaultResolutions.concat([0.25, 0.1]);
```

Comment:
I'm not sure to undrstand why this doesn't work the `concat` doesn't affect the defaultResolutions array.